### ISSUE TITLE
SVG size / scale is hard to predict and not as expected

### ISSUE THREAD

**Issue by hmeine** [Weasy666/bevy_svg#18]
I am trying out bevy_svg and may be missing quite some knowledge or assumptions bevy_svg or bevy make, so my issue is a mixture of a support request and a hint at what might be good to add in the documentation.

I have successfully loaded an SVG file and would now like to "scale it properly" on screen. The asset should eventually cover a certain percentage of the window size, I think. Hence, I have written a system that provides debug output of the documented Svg properties `size` and `view_box`:
```rs
fn svg_loaded_system(mut ev_asset: EventReader<AssetEvent<Svg>>, assets: Res<Assets<Svg>>) {
    for ev in ev_asset.iter() {
        if let AssetEvent::Created { handle } = ev {
            let svg = assets.get(&handle.clone()).unwrap();
            println!("svg loaded, size {} and view box {:?}", svg.size, svg.view_box);
        }
    }
}
```

I guessed that size would be the size of the view_box, but I got:
```
svg loaded, size [222.78255, 323.556] and view box ViewBox { x: 0.0, y: 0.0, w: 167.0869141, h: 242.6669922 }
```

The view box matches what I see in inkscape, so I would also have expected that to be the size on screen. With the default `Camera2dBundle`, I get a size of 335x486 pixels based on a Gimp measurement from a screenshot, which fits well the retina display rendering at a x2 scale.

So, the only question is where the size [222.78255, 323.556] comes from?  The current documentation of `Svg.size` is not helpful at all ("Size of the SVG") – I would suggest not to repeat what's already clear from the name itself, but to specify units, measurement details, whatever context this may be coming from, used in, or intended for.

**Comment by hmeine** [comment]
No, I think I understand more now that I have studied the file again in Inkscape, and looked at the SVG spec.

The **document size in pixels** is [222.78255, 323.556], whereas the view box is in document units (ratio 3:4, apparently configurable).  Interestingly, the size on screen does not seem to be based on the pixel size, although that would look like the more natural default, don't you agree?

**Comment by hmeine** [comment]
In the meantime, I have also found that two SVG files that should have the same size do not with bevy_svg. In Inskcape, the sizes in mm *and* px are the same, only the "scale factor" between the internally used SVG coordinates and mm displayed in the document properties differs.

The worst case, however, is an SVG file I came across in which different elements seem to get different scales applied, completely breaking the overall appearance.  So I guess applying the proper scales is not optional, but necessary in order to ensure correct rendering of all SVG assets.

**Comment by StarArawn** [comment]
Here is some calculations for resizing the SVG:
```rust
let new_height = (svg.view_box.h as f32 / svg.view_box.w as f32) * new_width;
let svg_scale_x = new_width / svg.view_box.w as f32;
let svg_scale_y = new_height / svg.view_box.h as f32;
```
Note: The calculated `svg_scale_x` and `svg_scale_y` are transform scales.

**Comment by Weasy666** [comment]
Hopefully fixed by #35 . If not, just tell me and we can reopen this issue.
